common/error_handler.go

```diff
@@ -16,8 +19,8 @@ const (
 
 func HandleError(ctx context.Context, w http.ResponseWriter, kind Kind, message string, cause error, httpErrorMapper func(context.Context, error) *HTTPError) {
 	err := CreateError(ctx, kind, message, cause)
-	logEntry := GetLogEntryFromContext(ctx)
-	logEntry.Error(err)
+	bytes, _ := json.Marshal(err)
```

Comment:
Is json marshalling necessary? What does the standard logger do if you just pass an error?

Reply:
The look hook will get empty message if we don't do json marshalling.
eg: in `simple_test.go` line 212, 223, 235

`require.Equal(t, "{\"Kind\":2,\"Message\":\"foo\",\"Cause\":null}", hook.LastEntry().Message)`

Reply:
The log entry should contain the error within its `Data` field. 

The real question about this refactor is: does the logger log out the error in a meaningful way? `log.Error(ctx, err)` should be sufficient to log the error. If you want to test what the logger actually logs then you can add the `SetOutput` config when you're configuring the logger.